fcsa_service_request-1.0.0-resolved.json

```diff
@@ -0,0 +1,571 @@
+                }
+              }
+            }
+          },
+          "400":{
+            "description": "Bad Request",
+            "headers": {
+              "Set-Cookie": {
+                "description": "Pass this cookie back in subsequent requests.\n",
+                "style": "simple",
+                "explode": false,
+                "schema": {
+                  "$ref": "#/components/schemas/Error_Schema"
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "cookieAuth": []
+          }
+        ]
+      }
+    },
+    "/zsreq/session": {
+      "get": {
+        "tags": [
+          "auth_group"
+        ],
+        "summary": "Returns a new session identifier, to be used in subsequent requests as the value for the X-Session header.\n",
+        "parameters": [
+          {
+            "name": "cookie",
+            "in": "query",
+            "description": "The MYSAPSSO2 authentication cookie. See /coct/api/zcur-guest/login path for details on how to retrieve it.\n",
+            "required": true,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "type": "string",
+              "format": "uuid"
+            }
+          },
+          {
+            "name": "X-Service",
+            "in": "header",
+            "description": "This service retrieves a session identifier, in the form of a 32-character Globally Unique Identifier (GUID), to be used for the X-Service parameter.\n",
+            "required": true,
+            "style": "simple",
+            "explode": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "X-Session",
+            "in": "header",
+            "description": "Retrieves a new session identifier for the X-Session header parameter.\n",
+            "required": true,
+            "style": "simple",
+            "explode": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ok",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Session_Schema"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Error in retrieving all type codes and descriptions. (Internal)",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error_Schema"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/zsreq/config/subtypes": {
+      "get": {
+        "tags": [
+          "service_request_group"
+        ],
+        "summary": "This service gets the full range of subtype codes and descriptions for all service groups.\n",
```

Comment:
/nitpick

Calling request categories "codes" is a bit of City specific jargon - I think most people would expect them to be called request types. I suggest something like "Returns the full set of request types or codes, and their descriptions for the City's Service Requests"